suppress.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
remove the suppressions file, we need to resolve them.
we will add a new one when we find issues.

Reply:
Deleted suppress.xml.